server/app.js

```diff
@@ -3,11 +3,38 @@ import express, { json, urlencoded } from "express";
 import { join } from "path";
 import cookieParser from "cookie-parser";
 import logger from "morgan";
+import mongoose from "mongoose";
+// var mongoose = require('mongoose');
 
 import indexRouter from "./routes/index";
 import pingRouter from "./routes/ping";
+let  users= require('./schemas/users');
 
 var app = express();
+mongoose.connect('mongodb://localhost/bakeryshop', {useNewUrlParser: true});
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+      console.log("you are connected");
+      users.find({ 'id': 1 }, function (err, found) {
```

Comment:
This is good for testing purposes, but let's remove the user creation here - we can create a seeding script however (in a separate file), that you can call from terminal to add a bunch of data